lib/integration/scenarios/Test/Integration/Scenario/API/Voting.hs

```diff
@@ -227,10 +227,16 @@ spec = describe "VOTING_TRANSACTIONS" $ do
             }|]
         rTx1 <- request @(ApiConstructTransaction n) ctx
             (Link.createUnsignedTransaction @'Shelley src) Default delegationJoin
+
```

Comment:
This test is 250 lines long, it's pretty hard to follow and understand what's going exactly. It seems the change introduced to do not change the logic, only the structure of the results?